packages/mui-base/src/Slider/index.ts

```diff
@@ -1,4 +1,30 @@
-'use client';
-export { Slider } from './Slider';
-export * from './Slider.types';
-export * from './sliderClasses';
+export { Slider as Root } from './Root/SliderRoot';
+export {
+  SliderRootOwnerState as SliderOwnerState,
+  SliderRootProps as RootProps,
+  UseSliderParameters,
+  UseSliderReturnValue,
+  SliderContextValue,
+  SliderProviderValue,
+  SliderThumbMetadata,
+  Axis,
+} from './Root/SliderRoot.types';
+export { useSliderRoot } from './Root/useSliderRoot';
+export * from './Root/SliderProvider';
+
+export { SliderThumb as Thumb } from './SliderThumb/SliderThumb';
+export type {
+  SliderThumbOwnerState as ThumbOwnerState,
+  SliderThumbProps as ThumbProps,
+  UseSliderThumbParameters,
+  UseSliderThumbReturnValue,
+} from './SliderThumb/SliderThumb.types';
+export { useSliderThumb } from './SliderThumb/useSliderThumb';
+
+export { SliderTrack as Track } from './SliderTrack/SliderTrack';
+export type { SliderTrackProps as TrackProps } from './SliderTrack/SliderTrack.types';
+export { useSliderTrack } from './SliderTrack/useSliderTrack';
```

Comment:
Parameters and return values of hooks should be exported as well

Reply:
Updated!